app/src/test/java/com/distillery/interview/ui/DailyWeatherViewModelTests.kt

```diff
@@ -55,8 +54,8 @@ class DailyWeatherViewModelTests {
     @Test
     fun getDailyWeather_returnErrorResponse() = runBlockingTest {
         //GIVEN
-        val mockError = Result.Error()
-        whenever(weatherRepository.getDailyWeather()).thenReturn(mockError)
+        val mockError = RuntimeException()
```

Comment:
We can simulate an `IOException` since is more like to happen than a `RuntimeException`.